scripts/mcstatus.coffee

```diff
@@ -0,0 +1,48 @@
+# Description:
+#   Report the status of a Minecraft server
+#   Uses http://api.syfaro.net/
```

Comment:
If I had more time to devote to hobby programming I'd probably do that, because adding an external dependency of unknown reliability is kinda meh to me.